model_incoming_transactions.go

```diff
@@ -344,12 +341,12 @@ func processIncomingTransaction(ctx context.Context, logClient zLogger.GormLogge
 	if transaction == nil {
 		// Create the new transaction model
 		if transaction, err = newTransactionFromIncomingTransaction(incomingTx); err != nil {
-			logClient.Error(ctx, fmt.Sprintf("processIncomingTransaction(): newTransactionFromIncomingTransaction() for %s failed. Reason: %s", incomingTx.ID, err))
+			logClient.Error().Msgf("processIncomingTransaction(): newTransactionFromIncomingTransaction() for %s failed. Reason: %s", incomingTx.ID, err)
```

Comment:
Can be added to `Str(processIncomingTransaction(): newTransactionFromIncomingTransaction())'`

Reply:
We shouldn't put this kind on information into context, because we probably won't filter log by them.

But now, when we can configure logger to use "origin"/"caller"/"stack trace" this info in message is redundant